src/dsp_tools/fast_xmlupload/process_files.py

```diff
@@ -102,7 +103,28 @@ def _process_files_in_parallel(
 
     orig_filepath_2_uuid: list[tuple[Path, Optional[Path]]] = []
     for processed in as_completed(processing_jobs):
-        orig_filepath_2_uuid.append(processed.result())
+        try:
+            orig_filepath_2_uuid.append(processed.result())
```

Comment:
maybe, instead of having this as a one-liner, you could first get it, then log some indication of what happened, and finally append it?